### ISSUE TITLE
When will open the training set?

### ISSUE THREAD

**Issue by Quan-zzx** [KivingY/ReCon1M#1]
Hello, I am very interested in your work
When will  open training set？ Thanks！

**Comment by ipinid613** [comment]
I find it that the author uploaded **only validation set of around 3000+ images.**
Hope the author open full dataset so that researchers can contribute to the community!
